src/api/webProfileApiClient.ts

```diff
@@ -23,6 +28,16 @@ const withJwtBearer: WithDefaultsT<'bearerAuth'> = (wrappedOperation: any) => (p
   });
 };
 
+// TODO -> SIW 918: remove this in production to use the right token and use 
```

Comment:
is this comment still valid? I'm asking since we're going to deploy this into production